src/campsite-info/campsite-info.component.tsx

```diff
@@ -1,21 +1,28 @@
 import React, { FunctionComponent } from 'react';
 import { CampHeader } from './components/camp-header';
 import { CampImages } from './components/camp-images';
+import { CampMap } from './components/camp-map';
 import { Box } from '@mui/material';
-import { graphicsStyles, mapStyles, imageStyles } from './campsite-info.styles'
+import { graphicsStyles, mapStyles, imageStyles } from './campsite-info.styles';
 
 export const CampsiteInfo: FunctionComponent = () => {
-    return (
-        <div>
-            <CampHeader region="region" park="park" zone="zone" site="site" logout={() => {}}/> {/* eslint-disable-line @typescript-eslint/no-empty-function */}
-            <Box sx={graphicsStyles} component="div">
-                <Box sx={mapStyles} component="div">
-                    <div /> {/* map goes here */}
-                </Box>
-                <Box sx={imageStyles} component="div">
-                    <CampImages />
-                </Box>
-            </Box>            
-        </div>
-    );
+  return (
+    <div>
+      <CampHeader
+              region="region"
+              park="park"
+              zone="zone"
+              site="site"
+              logout={() => {undefined}}
```

Comment:
this syntax is a bit odd: are you meaning to return undefined from the function? If so, this is redundant and you can just do this: `() => {}`

Reply:
This was a while ago, but I think I recall why I did this:
- Prettier formatted the code so that everything was on its own line (which is more readable)
- There used to be an es-lint ignore statement after the angle bracket, which we caused a lot of grief for us to figure out
- The statement doesn't work when it isn't on the same line as the empty arrow function
- TypeScript won't compile with an empty arrow function

I see what you're saying though. I think removing the curly braces around undefined will optimise readability and functionality here.